crates/cargo-test-support/src/registry.rs

```diff
@@ -502,15 +517,26 @@ impl Package {
     }
 
     fn append_manifest<W: Write>(&self, ar: &mut Builder<W>) {
-        let mut manifest = format!(
+        let mut manifest = String::new();
+
+        if !self.cargo_features.is_empty() {
+            manifest.push_str(&format!("cargo-features = {:?}\n\n", self.cargo_features));
```

Comment:
```suggestion
            manifest.push_str(&format!(
                "cargo-features = {}\n\n",
                toml::to_string(&self.cargo_features).unwrap()
            ));
```

Although it is unlikely to ever change, I feel uncomfortable using debug formatting like this.  (This will require adding `toml = "0.5"` to `cargo-test-support/Cargo.toml`.)